tests/unit_tests/audit_log/test_audit_column.py

```diff
@@ -0,0 +1,18 @@
+from exasol.analytics.audit.audit import BaseAuditColumns
+from exasol.analytics.schema import Column
+
+
+def test_docstring():
```

Comment:
what does this test try to achieve

Reply:
or is simply the name weird

Reply:
I think the test is obsolete now. 
Please see next push, removing the test.